packages/config-loader/src/loader.ts

```diff
@@ -126,8 +126,16 @@ export async function loadConfig(
     .filter((e): e is { url: string } => e.hasOwnProperty('url'))
     .map(configTarget => configTarget.url);
 
-  if (remote === undefined && configUrls.length > 0) {
-    throw new Error(`Remote config detected but this feature is turned off`);
+  if (remote === undefined) {
+    if (configUrls.length > 0) {
+      throw new Error(
+        `Remote config detected but this feature is turned off. Please enable by passing remote option in loadBackendConfig() call inside packages/backend/src/index.ts. See https://backstage.io/docs/conf/writing#configuration-files for detailed info.`,
```

Comment:
`loadBackendConfig` isn't really the only place where this is called, so best keep it a bit more general than that. Perhaps something along the lines of "please make sure you are passing the `remote` option when loading the configuration"?